sql/2020/07_SEO/pages_wpt_bodies_by_device_and_word_count.sql

```diff
@@ -0,0 +1,43 @@
+#standardSQL
+# page wpt_bodies metrics grouped by device and word count
+
+# helper to create percent fields
+CREATE TEMP FUNCTION AS_PERCENT (freq FLOAT64, total FLOAT64) RETURNS FLOAT64 AS (
+  ROUND(SAFE_DIVIDE(freq, total), 4)
+);
+
+# returns all the data we need from _wpt_bodies
+CREATE TEMPORARY FUNCTION get_wpt_bodies_info(wpt_bodies_string STRING)
+RETURNS STRUCT<
+    page_word_count INT64
+
+> LANGUAGE js AS '''
+var result = {};
+try {
+    var wpt_bodies = JSON.parse(wpt_bodies_string);
+
+    if (Array.isArray(wpt_bodies) || typeof wpt_bodies != 'object') return result;
+
+    if (wpt_bodies.visible_words) {
+      result.page_word_count = wpt_bodies.visible_words.rendered;
+
+    }
+
+} catch (e) {}
+return result;
+''';
+
+SELECT
+client,
+COUNT(0) AS total, 
+wpt_bodies_info.page_word_count as words_count
```

Comment:
Ditto on percentiles

Reply:
This was from last years query, and in my opinion not a very accurate value.  The pages_wpt_bodies_by_device_and_percentile.sql contains new metrics for visible words. 

I'll drop this query.